uint8 axis
float32 traj_accel_limit
float32 traj_decel_limit
---
bool success